### PR TITLE
Cut 22.3.1

### PR DESCRIPTION
**Pull request by jaimergp**
Add a changelog too to keep track of the shipped changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+
+## [22.3.1] - 2022.03.23
```
Comment (jezdez) [inline CHANGELOG.md:28]:
```suggestion
## [22.3.1] - 2022-03-23
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+### Fixed
+
+* Make sure `noarch` packages get reinstalled if Python version changed (#26)
+* Accept star-only version specs (e.g. `libblas=*=*mkl`) and fix support for `channel::package` syntax (#25)
+* Enable support for authenticated channels (#23)
+
+## [22.3.0] - 2022.03.09
```
Comment (jezdez) [inline CHANGELOG.md:36]:
```suggestion
## [22.3.0] - 2022-03-09
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+_First public release_
+
+## [22.2.0] - 2022.02.01
```
Comment (jezdez) [inline CHANGELOG.md:40]:
```suggestion
## [22.2.0] - 2022-02-01
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+> The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+> and this project adheres to [calendar versioning](https://calver.org/).
```
Comment (jezdez) [inline CHANGELOG.md:6]:
```suggestion
> and this project adheres to [calendar versioning](https://calver.org/) in the `YY.M.MICRO`format.
```

**Review by jezdez — APPROVED** [review]
Just a few nits to err on the side of ISO standards. Thank you @jaimergp!

**Comment by jezdez** [comment]
BTW I also created the GitHub releases for the past tags.

**Comment by jaimergp** [comment]
Ready!

**Pull request merged**